e2e/playwright/tests/meditor.spec.ts

```diff
@@ -0,0 +1,506 @@
+  "000024": [noContributorSelector, noLicenseSelector],
+  "000028": [noContributorSelector, noLicenseSelector],
+  "000030": [noContributorSelector, noLicenseSelector],
+  "000031": [noContributorSelector, noLicenseSelector],
+  "000033": [noContributorSelector, noLicenseSelector],
+  "000038": [noContributorSelector, noLicenseSelector],
+  "000042": [noContributorSelector, noLicenseSelector],
+  "000046": [noContributorSelector, noLicenseSelector],
+  "000047": [noContributorSelector, noLicenseSelector],
+  "000050": [noContributorSelector, noLicenseSelector],
+  "000051": [noContributorSelector, noLicenseSelector],
+  "000052": [noContributorSelector, noLicenseSelector],
+  "000060": [noLicenseSelector],
+  "000063": [noContributorSelector, noLicenseSelector],
+  "000068": [noContributorSelector, noLicenseSelector],
+  "000071": [noContributorSelector],
+  "000072": [noContributorSelector, noLicenseSelector],
+  "000106": [invalidLicenseSelector],
+  "000107": [invalidLicenseSelector, noContributorSelector],
+  "000112": [noContributorSelector, noLicenseSelector],
+  "000113": [noContributorSelector, noLicenseSelector],
+  "000116": [noLicenseSelector],
+  "000120": [noLicenseSelector],
+  "000124": [noLicenseSelector],
+  "000131": [noLicenseSelector],
+  "000132": [noLicenseSelector],
+  "000133": [noLicenseSelector],
+  "000134": [noLicenseSelector],
+  "000135": [noLicenseSelector],
+  "000136": [noLicenseSelector],
+  "000137": [noLicenseSelector],
+  "000141": [invalidLicenseSelector],
+  "000144": [noLicenseSelector],
+  "000145": [noLicenseSelector],
+  "000146": [noLicenseSelector],
+  "000150": [noLicenseSelector],
+  "000151": [noLicenseSelector],
+  "000155": [noLicenseSelector],
+  "000157": [noLicenseSelector],
+  "000158": [noLicenseSelector],
+  "000160": [noLicenseSelector],
+  "000162": [noLicenseSelector],
+  "000164": [noLicenseSelector],
+  "000170": [noLicenseSelector],
+  "000171": [noLicenseSelector],
+  "000208": [noLicenseSelector],
+  "000210": [noLicenseSelector],
+  "000214": [noLicenseSelector],
+  "000216": [noLicenseSelector],
+  "000225": [noLicenseSelector],
+  "000227": [noLicenseSelector],
+  "000229": [noLicenseSelector],
+  "000255": [noLicenseSelector],
+  "000290": [noLicenseSelector],
+  "000362": [titleTooLongSelector],
+  "000449": [noLicenseSelector],
+  "000534": [noContributorSelector],
+};
+
+test.describe("Test meditor validation errors", async () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(clientUrl);
+    await page.getByRole("button", { name: "Log In with GitHub" }).click();
+    await page.getByPlaceholder("Email address").click();
+    await page.getByPlaceholder("Email address").fill("admin@kitware.com");
+    await page.getByPlaceholder("Password").click();
+    await page.getByPlaceholder("Password").fill("password");
+    await page.getByRole("button", { name: "Sign In " }).click();
+    await page.waitForLoadState("networkidle");
```

Comment:
This is cool! However, the [docs](https://playwright.dev/docs/api/class-frame#frame-wait-for-load-state) state

> 'networkidle' - DISCOURAGED wait until there are no network connections for at least 500 ms. Don't use this method for testing, rely on web assertions to assess readiness instead.

What's our justification for ignoring their (strongly worded) recommendation?